src/cli.coffee

```diff
@@ -15,18 +9,25 @@ options = docopt """
   """
 
 
-generateConfig = ->
-  break for extension in extensions when options[extension]
+{docopt} = require 'docopt'
+options = docopt usage
+
+
+generateConfig = (extension) ->
+  asyncHandlers = require 'async-handlers'
+  fsExtra = require 'fs-extra'
```

Comment:
I'm thinking about a linter for this type of thing. One that checks that variable are declared in their minimal scope. I think it would be better in JSHint then coffeelint and thus I'm interested in trying out https://github.com/Clever/coffee-jshint